corehq/tabs/templatetags/menu_tags.py

```diff
@@ -87,9 +87,11 @@ def render(self, context):
         # set the context variable in the highest scope so it can be used in
         # other blocks
         context.dicts[0]['active_tab'] = active_tab
-        return mark_safe(render_to_string('tabs/menu_main.html', {
+        flat = context.flatten()
+        flat.update({
             'tabs': visible_tabs,
-        }))
+        })
+        return mark_safe(render_to_string('tabs/menu_main.html', flat))
```

Comment:
so we were just never including _any_ context here (except for the `tabs`)? good catch!

Reply:
indeed :-/